server/action_creators.js

```diff
@@ -20,6 +26,23 @@ export function updateOrder(orderId, order) {
   };
 }
 
+export function getInitialState() {
+  return (dispatch, getState) => {
+    dispatch(isLoading());
+    const getItems = () => Item.find().sort('category');
+    const getDiscounts = () => Discount.find();
```

Comment:
Am treating it like a master list of items so would make sense to put it in an initial state. Also, if it's not loaded here, say user is on the order details for an order that's ready for bill and the app crashes, and user refreshes, discounts would be empty since it's not loaded in the initial state.